be/src/olap/tablet.cpp

```diff
@@ -2057,7 +2057,7 @@ Status Tablet::cooldown() {
 
     if (_cooldown_replica_id == replica_id()) {
         // this replica is cooldown replica
-        RETURN_IF_ERROR(_cooldown_data());
+        RETURN_IF_ERROR(_cooldown_data(rowset));
```

Comment:
```suggestion
        RETURN_IF_ERROR(_cooldown_data(std::move(rowset)));
```suggestion
        RETURN_IF_ERROR(_cooldown_data(rowset));
```
```